extension-impl/bootstrap-bolt/pom.xml

```diff
@@ -22,6 +22,22 @@
             <artifactId>sofa-rpc-extension-common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+         <dependency>
+            <groupId>com.alipay.sofa</groupId>
+            <artifactId>sofa-rpc-remoting-bolt</artifactId>
+            <version>${project.parent.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
it's unnecessary to add `sofa-rpc-remoting-bolt` dependency.

try to add test resource files, refer https://github.com/alipay/sofa-rpc/tree/master/extension-impl/bootstrap-dubbo/src/test/resources

Reply:
done.